pkg/security/registry/podspecreview/registry.go

```diff
@@ -0,0 +1,31 @@
+package podspecreview
+
+import (
+	api "github.com/openshift/origin/pkg/security/api"
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/runtime"
+)
+
+type Registry interface {
```

Comment:
I don't think we need to create these interfaces anymore.  Pretty much everything calls back out to the API.